### PR TITLE
MINOR: Update `config/consumer.properties` to have new consumer properties

### PR DESCRIPTION
**Pull request by omkreddy**

### CODE REVIEW

**Comment by ijuma** [comment]
Thanks for the PR. We should probably add a note to `upgrade.html` in case people were relying on that file.

File: config/consumer.properties [hunk 1]
```diff
@@ -12,18 +12,15 @@
...
+# format: host1:port1,host2:port2 ...
+bootstrap.servers=localhost:9092
 
 #consumer group id
 group.id=test-consumer-group
 
-#consumer timeout
-#consumer.timeout.ms=5000
+#What to do when there is no initial offset in Kafka or if the current
+#offset does not exist any more on the server: latest, earliest, none
```
Comment (ijuma) [inline config/consumer.properties]: Space missing after #

**Comment by omkreddy** [comment]
@ijuma  assuming this can be merged to 1.0.0, added a note to 1.0.0 docs.  updated the PR. Pl check.

**Review by ijuma — APPROVED** [review]
Thanks for the PR, LGTM. Will merge it to trunk and 1.0 since I believe we'll probably need another RC given issues that have already been found. cc @guozhangwang

**Comment by guozhangwang** [comment]
@ijuma ack.

File: config/producer.properties [hunk 2]
```diff
@@ -12,7 +12,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-# see kafka.producer.ProducerConfig for more details
+# see org.apache.kafka.clients.producer for more details
```
Comment (ijuma) [inline config/producer.properties:15]: I missed the first time around, but `ProducerConfig` is missing.

Comment (omkreddy) [reply config/producer.properties:15]: Ahh.sorry for this..raised another PR #4070

**Pull request closed without merging**